### PR TITLE
xarray

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @jhamman.

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @shoyer

File: recipes/xarray/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set version = "0.7.1" %}
```
Comment (shoyer) [inline recipes/xarray/meta.yaml]: 0.7.2 is out now

**Comment by ocefpaf** [comment]
@shoyer let me know if you want me to add your name as a maintainer of this recipe.

**Comment by pelson** [comment]
:+1: once CI passes.

**Comment by shoyer** [comment]
@ocefpaf I'm happy to consult on the recipes for any of my projects, but I'm not likely to be the best maintainer since I'm not using conda on an everyday basis anymore.

**Comment by ocefpaf** [comment]
> @ocefpaf I'm happy to consult on the recipes for any of my projects, but I'm not likely to be the best maintainer since I'm not using conda on an everyday basis anymore.

I should explain that by adding your name as a maintainer you will adim powers in the feedstocks repository. (Place where the recipe will live after merging this PR.)

You don't really have any obligations but you could, for example, send a version update and merge it yourself.

**Comment by shoyer** [comment]
OK, it could be nice to have that ability.

**Comment by ocefpaf** [comment]
> OK, it could be nice to have that ability.

I added you to `h5netcdf` and `xarray`, but that a look at the other feedstocks in [conda-forge](https://github.com/conda-forge/) and let me know if you want to be maintainer of any other package. I suggest `cyordereddict`, `pynio` and its depedencies.

PS: You can add yourself and/or make changes in any feedstocks via PRs. Once the PR is merged you'll get the commit rights.

**Comment by pelson** [comment]
> since I'm not using conda on an everyday basis anymore.

Curious. Are you using binary wheels or just your OS?

**Comment by shoyer** [comment]
Google has its own build system that it uses for everything:
https://github.com/bazelbuild/bazel

**Pull request merged**